lib/Supervisor.js

```diff
@@ -201,7 +201,23 @@ function run(
 
                 xml.testsuite.testcase = xml.testsuite.testcase.concat(values);
 
-                console.log(XmlBuilder.create(xml).end());
+                // The XmlBuilder by default does not remove characters that are
+                // invalid in XML, like backspaces. However, we can pass it an
+                // `invalidCharReplacement` option to tell it how to handle
+                // those characters, rather than crashing. In an attempt to
+                // retain useful information in the output, we try and output a
+                // hex-encoded unicode codepoint for the invalid character. For
+                // example, a backspace (`\u{08}` in Elm) will be output as
```

Comment:
```suggestion
                // example, a backspace (`\u{0008}` in Elm) will be output as
```

Reply:
Resolved in https://github.com/rtfeldman/node-test-runner/pull/625/commits/41e357c03a66b2ee872d5931d0c5640139a6b3aa